packages/server/api/webscrape.py

```diff
@@ -3,32 +3,63 @@
 import re
```

Comment:
General comments:

1. I'm starting to see the disadvantages of using multiple languages. We have our models currently defined in Mongoose Schemas. The options I see moving forward:
    1.  Create exact copies of the schemas in Mongoengine (or something similar) which means we now have **two sources of truth** for our data model. Not good.
    1. Create our own JSON format for these schemas and parse those into Mongoose and Mongoengine schemas. Also not good.
    1. Go without schema validation on the Mongo wrapper side and say fuck it. Really not good.
    1. Implement schema validation at the mongodb instance level. Not easy to version this, but probably the best option so far as it gives us one source of truth.
    1. Use JS or python for all backend code, means we share models between functions (best option), we won't be learning python (not great for some), but faster speed of development (good)

Reply:
Also, this is weird now: I think there's some room for confusion here. This endpoint scrapes the data, but then does it call `/graphql` to insert the doc? If not then we have two sources of truth again. But it we do, then we'll have increased latency due to two API calls

Reply:
Ok, so regarding this, this webcraping services' sole function is to return an object to the front end as an editable form for the user to edit and then save. Once they save the form, that is when we will call thee /graphql endpoint to POST the recipe, meaning that we will only require mongoose scehmas on the JS side.

Reply:
nice, I like that separation of concerns